src/components/progress-report/migrations/UpdateExplanationReasons.migration.ts

```diff
@@ -0,0 +1,42 @@
+import { node, relation } from 'cypher-query-builder';
+import { BaseMigration, Migration } from '~/core';
+import { ACTIVE } from '~/core/database/query';
+
+@Migration('2023-03-10T17:00:00')
+export class UpdateExplanationReasonsMigration extends BaseMigration {
+  private readonly findAndReplaceReason = async (
+    oldReason: string,
+    newReason: string,
+  ) => {
+    await this.db
+      .query()
+      .match([
+        node('node', 'ProgressReport'),
+        relation('out', '', 'varianceExplanation', ACTIVE),
+        node('varianceExplanation', 'ProgressReportVarianceExplanation'),
```

Comment:
None of this match is needed. The explanation label is sufficient. 
```suggestion
        node('', 'ProgressReportVarianceExplanation'),
```

Reply:
Oh, so the starting point can be whatever as long as it's a node?

Reply:
Correct and also order does not matter.